lib/Doctrine/Persistence/Mapping/Driver/AnnotationDriver.php

```diff
@@ -66,7 +66,7 @@ abstract class AnnotationDriver implements MappingDriver
     /**
      * Name of the entity annotations as keys.
      *
-     * @var string[]
+     * @var array<class-string, mixed>
```

Comment:
Looks like it was wrong, I used `mixed` because looking at its use:
- `doctrine/orm` the values are `int`: https://github.com/doctrine/orm/blob/f8fa0fe0694ee661bbcc18406475fdb1749b0be1/lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php#L52-L55
- `doctrine/couchdb-odm` are `bool` : https://github.com/doctrine/couchdb-odm/blob/9bf3d8ed9a649d91b6a66aa439b05a8a1f1ef7fd/lib/Doctrine/ODM/CouchDB/Mapping/Driver/AnnotationDriver.php#L24-L28

And here in tests we're using `bool`, I'll change it to `array<class-string, int|bool>`.